.github/workflows/release.yaml

```diff
@@ -28,6 +28,8 @@ jobs:
 
       - name: Verify version
         uses: home-assistant/actions/helpers/verify-version@master
+        with:
+          metadata-file: pyproject.toml
```

Comment:
Not too sure about that, why does not the action just handle it regardless of which of the meta file types that are used?

Reply:
Good point. I was overcomplicating things. Updated https://github.com/home-assistant/actions/pull/71 as well.